src/main/asciidoc/java/index.adoc

```diff
@@ -888,7 +888,7 @@ when the verticle is undeployed.
 
 === Verticle worker pool
 
-Verticle use the Vert.x worker pool for executing blocking actions, i.e `link:../../apidocs/io/vertx/core/Context.html#executeBlocking-io.vertx.core.Handler-boolean-io.vertx.core.Handler-[executeBlocking]` or
+Verticle uses the Vert.x worker pool for executing blocking actions, i.e `link:../../apidocs/io/vertx/core/Context.html#executeBlocking-io.vertx.core.Handler-boolean-io.vertx.core.Handler-[executeBlocking]` or
```

Comment:
I think that here _use_ is correct, we mean verticles in general or it would be _a_ verticle

Reply:
Ah yes, good catch.  I just changed it to "Verticles use the Vert.x worker pool" (so made "Verticle" plural).